src/utils.ts

```diff
@@ -37,13 +39,15 @@ export function wait(n: number): Promise<void> {
 	return new Promise((resolve) => setTimeout(resolve, n));
 }
 
-export function getTag(name: string): MediaType {
+export function getTag(name: string, isVideo?: boolean): MediaType {
 	return EP_REGEX.test(name)
 		? MediaType.EPISODE
 		: SEASON_REGEX.test(name)
 		? MediaType.SEASON
 		: MOVIE_REGEX.test(name)
 		? MediaType.MOVIE
+		: isVideo && ANIME_REGEX.test(name)
```

Comment:
it seems like most of the changes here are threading this isVideo boolean around. can it be calculated on the fly? what is it used for?

Reply:
Since this is dependent on searchee I updated it to pass that around instead. `getTag` and `reformatTitleForSearching` still takes a boolean though as it could be used multiple times by the caller.